### ISSUE TITLE
Property 'FACET_OF' missing in wikidata properties

### ISSUE THREAD

**Issue by karimvision** [openai/deeptype#10]
First of all, awesome work 👏  and thank you so much for sharing your knowledge, really appreciate it 😃 

it seems that the property **FACET_OF** is missing from the wikidata properties [here](https://github.com/openai/deeptype/blob/e23538f5fed4dd605dbf31cb89fdaa9c440b5127/wikidata_linker_utils_src/src/python/wikidata_linker_utils/wikidata_properties.py), this caused the error below while running 
_python3 extraction/fast_link_fixer.py ${DATA_DIR}wikidata ${DATA_DIR}${LANGUAGE}_trie ${DATA_DIR}${LANGUAGE}_trie_fixed_

inside extraction/full_preprocess.sh,
as a quick fix i just commented it out ([this line)](https://github.com/openai/deeptype/blob/master/extraction/fast_link_fixer.py#L273) and it works fine after that
```
load 'P31' ('instance of')
inverting relation 'P31' ('instance of')
load inverted 'P31' ('instance of')
load 'P279' ('subclass of')
inverting relation 'P279' ('subclass of')
load inverted 'P279' ('subclass of')
load 'P360' ('is a list of')
inverting relation 'P360' ('is a list of')
load inverted 'P360' ('is a list of')
load 'P361' ('part of')
inverting relation 'P361' ('part of')
load inverted 'P361' ('part of')
Traceback (most recent call last):
  File "extraction/fast_link_fixer.py", line 594, in <module>
    main()
  File "extraction/fast_link_fixer.py", line 469, in main
    num_category_link=8
  File "extraction/fast_link_fixer.py", line 273, in fix
    {"steps": [wprop.INSTANCE_OF, wprop.FACET_OF]},
AttributeError: module 'wikidata_linker_utils.wikidata_properties' has no attribute 'FACET_OF'
```

**Comment by galtay** [comment]
probably resolved by #9

**Comment by karimvision** [comment]
oh, it was a typo then, got it, thanks!, closing this issue now

**Comment by galtay** [comment]
updated fix to #12
